scripts/update-lib-version.bash

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+set -e
+
+############################################################
+# update all html files in the examples folder
+############################################################
+if [ $# -ne 1 ]; then
+	echo "Mandatory parameter <new_version> missing."
+	exit 1
+fi
+NEW_VERSION=$1
+echo "Updating examples to make them use demo@$NEW_VERSION"
+
+SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+EXAMPLES_DIRECTORY="$SCRIPT_DIRECTORY/../examples"
+pushd "$EXAMPLES_DIRECTORY" > /dev/null
+
+echo "Search for files in $(pwd)"
+# lines to update start with something like "<script src="../../demo/0.1.6/"
+sed -i -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
```

Comment:
To fix for MacOs
```suggestion
sed -i '' -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
```

Reply:
:warning: not working on Ubuntu 18.04
`Unable to read s#<script src="../../demo/[^/]+/#<script src="../../demo/0.14/#: no file or folder with this type` (literally translated from a French error message)

Reply:
work without space between `-i`and the argument value
`sed -i'' -E...` or `sed -i"" -E...`

Reply:
Doesn't work without space 😢

Reply:
I think if we want to use it on all platforms, we can't use **sed**.
https://unix.stackexchange.com/a/131940

Reply:
Ok, can we do the following?
- keep the existing script as it is because it works for Linux and Windows users
- add a comment just before the sed command for macOS user

In the future, this script will be run on Linux as part of the release automation. For now, it helps for the manual process.
Does that make sense @csouchet ?

Reply:
done via e516202